### PR TITLE
Bump docker to version 10.1.0

### PR DESCRIPTION
**Pull request by gurkanindibay**

### CODE REVIEW

File: postgres-12/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # This file is auto generated from it's template,
 # see citusdata/tools/packaging_automation/templates/docker/postgres-12/postgres-12.tmpl.dockerfile.
-FROM postgres:12.7
-ARG VERSION=10.0.4
+FROM postgres:12.6
```
Comment (hanefi) [inline postgres-12/Dockerfile:3]: Why the downgrade? Is this because of the template used in the scripts?

File: Dockerfile [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # This file is auto generated from it's template,
 # see citusdata/tools/packaging_automation/templates/docker/latest/latest.tmpl.dockerfile.
-FROM postgres:13.3
-ARG VERSION=10.0.4
+FROM postgres:13.2
```
Comment (hanefi) [inline Dockerfile:3]:
```suggestion
FROM postgres:13.3
```

Let's keep this as is, no need to downgrade

Comment (gurkanindibay) [reply Dockerfile]: Fixed

File: postgres-12/Dockerfile (hunk 1 above)
Comment (gurkanindibay) [reply postgres-12/Dockerfile]: Fixed

**Review by hanefi — APPROVED** [review]

**Pull request merged**